PlayCover/en.lproj/Localizable.strings

```diff
@@ -79,6 +88,11 @@
 "alert.legacyImport.subtitle" = "Legacy settings and keymapping files were detected. Would you like to convert these files to the new format?";
 "alert.wrongFileType.title" = "Wrong file type!";
 "alert.wrongFileType.subtitle" = "PlayCover can only install valid .ipa files.";
+"alert.uninstall.clearAppData" = "Clear app data";
```

Comment:
I think we can remove `app` from these strings since it is derived from context.